synchronizer/l1_syncstatus.go

```diff
@@ -60,7 +60,7 @@ func (s *syncStatus) reset(lastBlockStoreOnStateDB uint64) {
 	s.lastBlockStoreOnStateDB = lastBlockStoreOnStateDB
 	s.highestBlockRequested = lastBlockStoreOnStateDB
 	s.processingRanges = newLiveBlockRanges()
-	s.lastBlockOnL1 = invalidLastBlock
+	//s.lastBlockOnL1 = invalidLastBlock
```

Comment:
Isn't this code needed?

Reply:
I was thinking about that... previously I discard the lastBlockOnL1 after a reset to force to retrieve it again, but I think that is not needed, because the block if change is going to increase never decrease. So it would be retrieve as the regular flow of the code